src/qml/qgismobileapp.qml

```diff
@@ -526,6 +526,13 @@ ApplicationWindow {
 
       bgcolor: "#64B5F6"
 
+      onIconSourceChanged: {
+        if( state === "On" && !positionSource.position.latitudeValid )
+          displayToast( qsTr( "No position source available" ) )
+      }
+
+      property bool followActive: false
```

Comment:
What's this for?

Reply:
Removed it... It's a copy pasting leftover on the manual backport.